packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -266,7 +267,6 @@
   }
 
   &.mdc-textfield--disabled {
-    @include mdc-theme-prop(background-color, $mdc-textfield-light-background);
```

Comment:
I'm actually wondering whether this style (which is fully-opaque white) or the hard-coded one below is intended to be used for disabled textfield boxes... cc @amsheehan

Reply:
Confirmed that black 2% opacity is correct. Made it a variable for easier changes in the future.